modules/persistent-storage-csi-migration-overview-support-level.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-migration.adoc
+
+:_content-type: CONCEPT
+[id="persistent-storage-csi-migration-overview-support-level_{context}"]
+= Support level
+
+Certain in-tree volume plug-ins and their equivalent Container Storage Interface (CSI) driver are supported in Technology Preview (TP) status, whereas others are supported in General Availability (GA) status.
+
+:FeatureName: CSI automatic migration for some in-tree volume plug-ins/CSI driver pairs
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+The following table provides details about the support level of in-tree volume plug-ins/CSI driver pairs.
+
+.CSI automatic migration In-tree volume plug-ins/CSI driver pair support in {product-title}
+[cols=",^v,^v,^v width="100%",options="header"]
+|===
+|CSI drivers |Support level |CSI Auto migration enabled automatically?
```

Comment:
I would change "support level" to "Migration support level" in to order to avoid potential confusion on the CSI driver itself being TP.